etc/scripts/licenses/report_license_rules.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import io
+
+import click
+import csv
+
+from commoncode.cliutils import PluggableCommandLineOption
+from licensedcode.models import load_licenses
+from licensedcode.models import load_rules
+
+
+LICENSES_FIELDNAMES = [
+    'key', 'short_name', 'name', 'category', 'owner',  'text', 'words_count', 'notes',
```

Comment:
text is always going to be problematic... using the first `[:200]` characters is going to be enough